lib/net.c

```diff
@@ -185,14 +186,11 @@ ntuple_filter_add(uint8_t portid, uint32_t dst_ip,
 		.queue = queue_id,
 	};
 
-	ret = rte_eth_dev_filter_supported(portid, RTE_ETH_FILTER_NTUPLE);
-	if (ret < 0) {
-		RTE_LOG(ERR, PORT,
-			"Ntuple filter is not supported on port %hhu.\n",
-			portid);
-		ret = -1;
-		goto out;
-	}
+	RTE_VERIFY(rte_eth_dev_filter_supported(portid,
+		RTE_ETH_FILTER_NTUPLE) == 0);
+
+	if (!ipv4_configured)
+		goto ipv6;
 
 	if (dst_ip != 0) {
```

Comment:
Thanks to the new `if (!ipv4_configured) goto ipv6;` above, we can drop this `if` test, can't we?